src/diffusers/guiders/guider_utils.py

```diff
@@ -72,42 +97,52 @@ def set_state(self, step: int, num_inference_steps: int, timestep: torch.LongTen
         self._timestep = timestep
         self._count_prepared = 0
 
-    def set_input_fields(self, **kwargs: Dict[str, Union[str, Tuple[str, str]]]) -> None:
```

Comment:
Are we removing this method? 👀 

The `def _prepare_batch` has a reference to this method in an error:
https://github.com/huggingface/diffusers/blob/f072c64bf22c6a259d38435b19a8ef3eb67c96a8/src/diffusers/guiders/guider_utils.py#L187

Reply:
yes